src/ui/components/Timeline/Timeline.css

```diff
@@ -50,6 +50,8 @@
 }
 
 .timeline .progress-bar {
+  display: flex;
+  align-items: center;
```

Comment:
Just gonna suuuuper nit here and advocate for sorting styles when they are all of equal importance.

Reply:
well in this case just putting `align-items` first since it comes earlier in the alphabet. I usually either have a css linter do this for me or run `:sort` in vim.